### DISCUSSION TITLE
Filter box

### DISCUSSION THREAD

**Discussion by dsKanishk** [apache/superset#26165] [Q&A / Help] [answered]
I am trying to create a filter box, for some reason I do not see that as a chart option.
Although the preloaded dashboard has it, but we don't get to use (recreate with diif. dataset and save in diff dashboard) it.
I am aware that it works with certain chart type only I am choosing tables which it supports, still not able to create it.
Please help.
It would be much appreciated. 
Thanks.

**Comment by michael-s-molina** [comment, marked as answer]
Hi @dsKanishk. Filter boxes were deprecated in favor of native filters. Check https://superset.apache.org/docs/miscellaneous/native-filter-migration/

**Reply by dsKanishk** [reply]
Thanks!!
We're exploring ways to make filters visible, perhaps by incorporating dynamic headers that display the applied filters. This visibility is crucial for users receiving PDF or image reports. It's our initial approach, and any guidance or suggestions on achieving this would be greatly appreciated. Thank you!

**Reply by michael-s-molina** [reply]
I agree that you need to see which filters are applied when receiving a report. I'm going to ping some folks that have knowledge of this area @eschutho @yousoph @kasiazjc @zuzana-vej

**Reply by dsKanishk** [reply]
Thanks!
Much appreciated.

**Reply by vergenzt** [reply]
Hi @michael-s-molina -- I get a 404 for this URL. I couldn't find a new location after a few minutes of googling. Any idea where this doc lives now?

<img width="663" alt="Screenshot 2024-03-21 at 10 06 33 AM" src="https://github.com/apache/superset/assets/1725615/050d5832-fe90-4e06-abb8-7bc3dec20dd8">

**Reply by vergenzt** [reply]
Oh I see it was removed from main repository with https://github.com/apache/superset/commit/d9a3c3e1dd23eca113a799bb6a2fa4abddd4a76b#diff-c2f18db682d85966cf87713d299fc81653a58e4373f48f8d22e315e6ebdbcb0e as part of 4.0.

Any idea if older versions' docs are rendered anywhere?

**Reply by michael-s-molina** [reply]
@vergenzt The docs are not versioned currently but you can view that content [here](https://github.com/apache/superset/blob/3.1/docs/docs/miscellaneous/native-filter-migration.mdx).

**Reply by vergenzt** [reply]
Got it. Thanks! (I've opened a discussion at https://github.com/apache/superset/discussions/27604 to request that rendered docs be versioned 😉)
